### ISSUE TITLE
More Block: Read More link points to the part of the post where the block was added

### ISSUE THREAD

**Issue by davoraltman** [Automattic/wp-calypso#55980]
### Steps to Reproduce

1. Add a More block to a post
2. Go to the main Posts page and click on the Read More link
3. You'll arrive at the post but not at its beginning, but rather at the very place where the More block was added

Video of the issue: https://d.pr/i/bazYJs/AKSxUOQeqd

### What I expected to happen

The link should take me to the beginning of the post instead.

### What actually happened

I arrived at the place in the post where the More block is placed

### Context

This was first reported in 4269022-zen

### Operating System

macOS

### Browser

Chrome/Chromium

### Simple/Atomic

Simple, Atomic

### Reproducibility

Consistent

**Comment by nickpagz** [comment]
Is this possibly a version issue, or only atomic sites? I use the More block regularly on a personal site (Simple site) and it works as expected.
https://onepercent.blog/

I haven't done any more testing tbh, but in my configuration it works, at least up to the latest post on August 17th.

**Comment by davoraltman** [comment]
The video I added was from a Simple site and this can also be reproduced on an Atomic site - https://testingat957860922.wpcomstaging.com/blog/

It seems that there's a combination of specific themes/more block that causes this issue. Ixion theme is one of them.

**Comment by nickpagz** [comment]
Yep, I did another test and I was able to replicate on Ixion as well, but Balasana was ok, so definitely a compatibility issue with some themes.

**Comment by synora** [comment]
I'm able to reproduce this on Ixion, Hemingway Rewritten, Coutoire, and Independent Publisher 2. Works as expected on Balasana, Dalston, Maywood, and Stratford.
